package.json

```diff
@@ -2,18 +2,22 @@
   "name": "xterm-pty",
   "description": "A pty for xterm.js and Emscripten",
   "version": "0.10.2",
-  "main": "index.js",
+  "main": "out/index.js",
+  "module": "out/index.mjs",
```

Comment:
Do you need to bundle `index.mjs`? If needed, I would like to apply webpack to minify it as well.